QEfficient/exporter/export_hf_to_cloud_ai_100.py

```diff
@@ -7,30 +7,30 @@
 
 import os
 import shutil
-from typing import Optional, Tuple
+from typing import Optional, Tuple, Union
 
 import torch
-from huggingface_hub import login
-from transformers import AutoTokenizer
+from transformers import PreTrainedTokenizer, PreTrainedTokenizerFast
 
+import QEfficient
 from QEfficient.exporter.export_utils import export_onnx, fix_onnx_fp16, generate_input_files, run_model_on_ort
-from QEfficient.transformers.modeling_utils import transform
-from QEfficient.utils import hf_download
+from QEfficient.src._transformers.auto import QEFFAutoModelForCausalLM
```

Comment:
src also has transformers dir, and there's one outside too. 
Better to have unique names, unless there's a specific reason to keep it like this

Reply:
Will be taken up later.